### PR TITLE
Minor fixes in syntax and naming

### PR DESCRIPTION
**Pull request by tapishr**
- [x] Build passed
- [x] Integration tested
- [x] rustApp changed to dpu
- [x] try catch syntax changed in setup.ts
- [x] [PR for dpu for change in url endpoint](https://github.com/Alokit-Innovations/vibi-dpu/pull/10)

### Summary by CodeRabbit

- New Feature: Added a new `setupHandler` function in `setupHandler.ts` that handles HTTP requests and saves setup information in the database.
- Improvement: Introduced validation checks in `rustApp/setup.ts` to ensure the request body is properly formatted before processing. This prevents potential errors and enhances the robustness of the application.

### CODE REVIEW

File: pages/api/dpu/setup.ts
Comment (MuskanPaliwal) [inline pages/api/dpu/setup.ts:1]: Await per .catch() kar sakte hain?

**Review by MuskanPaliwal — APPROVED** [review]

File: pages/api/dpu/setup.ts
Comment (tapishr) [reply pages/api/dpu/setup.ts:1]: Yep, copying @avikalpg here when doing this. Tested this as well since I was also not sure how this would work

**Review by avikalpg — APPROVED** [review]
Looks good to me, although you will have to merge the `main` branch into this branch and push again.
Feel free to use your admin privileges to merge this branch once you have done that.

File: pages/api/dpu/setup.ts
Comment (tapishr) [reply pages/api/dpu/setup.ts:1]: Changed this part in PR #146

File: pages/api/dpu/setup.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    await Promise.all(allTopicPromises).then((values) => {
+        console.info("[setupHandler] All setup info saved succesfully...")
+        res.status(200).send("Ok");
+        return;
+    }).catch((error) => {
+        console.error("[setupHandler] Unable to save all setup info in db, error: ", error);
+        res.status(500).json({"error": "Unable to save setup info"});
+        return;
+    });
+}
```
[the remark this answers is not in the document]
Comment (MuskanPaliwal) [reply pages/api/dpu/setup.ts:26]: @tapishr this one?

**Review by MuskanPaliwal — CHANGES_REQUESTED** [review]
Just one comment to be addressed.

File: pages/api/dpu/setup.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (tapishr) [reply pages/api/dpu/setup.ts:26]: done

**Review by MuskanPaliwal — APPROVED** [review]

**Pull request merged**